defaults/main.yml

```diff
@@ -149,6 +149,13 @@ grafana_alerting:
 #  nodata_or_nullvalues: 'no_data'
 #  concurrent_render_limit: 5
 
+# Grafana logging configuration
+grafana_logging:
+# mode:
```

Comment:
I think it would be better to define `mode` variable as close to grafana configuration option as possible. Instead of defining a yaml list it might be better to use a simple string.